idseq_dag/steps/run_star.py

```diff
@@ -1,12 +1,219 @@
+import multiprocessing
+import os
+
 from idseq_dag.engine.pipeline_step import PipelineStep
 import idseq_dag.util.command as command
+import idseq_dag.util.log as log
 
 
 class PipelineStepRunStar(PipelineStep):
+    MAX_INPUT_READS = 75 * 1000 * 1000
```

Comment:
this can actually be obtained through self.additional_attributes["truncate_reads_to"]